practices/lockfile.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: lockfile
+stages:
+  - development
+short_description: A practice in our DX Scanner called LockFileIsPresent verifies if your source repository contains a lock file. You should always commit the lock file to your source repository.
```

Comment:
I wouldn't talk about DX Scanner and it's practice name here. I would rewrite the sentence to be more general.